### PR TITLE
Update language on miscellaneous pages (Playground, about, and blog)

### PR DESCRIPTION
**Pull request by malinatran**

### CODE REVIEW

**Review by davidvanbeveren — DISMISSED** [review]

**Review by davidvanbeveren — DISMISSED** [review]

File: src/pages/about/our-mission.tsx [hunk 1]
```diff
@@ -31,39 +31,38 @@ const OurMission: PageWithLayout = () => {
...
+            elevate our work, which is why we believe in gathering and analyzing
+            data, as well as making it accessible to others.
           </SubSection>
           <SubSection header="2. We need more competition, too.">
             We strongly believe competition is not only healthy but vital in
-            growing and improving our movements’ effectiveness. Competition puts
-            healthy pressure on other organizations and projects to keep
-            building, improving, and innovating – and gives them more data on
-            what works, and what doesn’t work.
+            improving our movement's effectiveness. Competition applies healthy
```
Comment (Headscracher) [inline src/pages/about/our-mission.tsx]:
```suggestion
            improving our movement&apos;s effectiveness. Competition applies healthy
```

**Review by Headscracher — CHANGES_REQUESTED** [review]

File: src/pages/about/our-mission.tsx (hunk 1 above)
Comment (davidvanbeveren) [reply src/pages/about/our-mission.tsx]: Can't we use ’ like on line 45? @Headscracher

Comment (Headscracher) [reply src/pages/about/our-mission.tsx]: That's also possible. It's currently mixed in the repository, so feel free to use both solutions

**Review by Headscracher — DISMISSED** [review]

File: src/pages/about/our-mission.tsx [hunk 2]
```diff
@@ -15,10 +15,10 @@ const OurMission: PageWithLayout = () => {
     <>
       <NextSeo title="Our Mission" />
       <FirstSubSection header="Our mission">
-        Our mission is to build new, data-driven, disruptive, and innovative
-        projects to help see an end to animal exploitation. We believe the
-        animal protection movement has five fundamental areas it can improve on
-        that we aim to solve through the work that we do.
+        Our mission is to build data-driven, disruptive, and innovative projects
+        to help see an end to animal exploitation. We believe the animal
+        protection movement has five fundamental area for improvement, ones that
```
Comment (malinatran) [inline src/pages/about/our-mission.tsx]:
```suggestion
        protection movement has five fundamental area for improvement, which
```

File: src/pages/about/our-mission.tsx [hunk 3]
```diff
@@ -31,39 +31,38 @@ const OurMission: PageWithLayout = () => {
...
           <SubSection header="1. We need more data in our movement.">
-            We can’t realistically track or be confident that the projects and
-            organizations out there right now are as effective as they can be.
-            We strongly believe this is the largest issue in growing our
-            movement, and so we aim to build, utilize, and gather data through
-            our projects and network – and then share that data.
+            To determine effectiveness in projects, campaigns, and our work as a
+            whole, meaningful data needs to be tracked and collected. We
+            strongly believe that a data-driven movement will accelerate and
+            elevate our work, which is why we believe in gathering and analyzing
```
Comment (malinatran) [inline src/pages/about/our-mission.tsx]:
```suggestion
            elevate our work, which is why we prioritize identifying, gathering, and analyzing
```

**Review by Headscracher — APPROVED** [review]

**Pull request merged**